R/ConceptCountsTable.R

```diff
@@ -0,0 +1,82 @@
+# Copyright 2022 Observational Health Data Sciences and Informatics
+#
+# This file is part of CohortDiagnostics
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#' createConceptCountsTable
+#' 
+#' @description Create a table containing the concept counts.
```

Comment:
As this function can save users time, this could be elaborated on somewhat (e.g. some example usage)

Reply:
I expanded a bit the description.